perfkitbenchmarker/network.py

```diff
@@ -102,6 +102,85 @@ def __repr__(self):
     return '%s(%r)' % (self.__class__, self.__dict__)
 
 
+class BaseVPNGW(object):
+  """An object representing the Base VPN GW."""
+  CLOUD = None
+
+  def __init__(self, zone=None, cidr=None):
+    """Initializes the BaseVPNGW.
+
+    Args:
+      zone: The zone in which to create the VPNGW.
+    """
+    self.zone = zone
+    self.cidr = cidr
+    self.require_target_to_init = False  # True if we need taget GW up front (AWS)
```

Comment:
Where is this set to True for AWS?

Reply:
AWS is a future PR, but it's set in the [aws vpn gateway](https://github.com/SMU-ATT-Center-for-Virtualization/PerfKitBenchmarker/blob/346e5a671247d016a3b68588326891f6df4e895a/perfkitbenchmarker/providers/aws/aws_network.py#L903).